src/pages/index.tsx

```diff
@@ -80,9 +93,45 @@ const HomePage: NextPage = () => {
     updateFeatureModalShown();
   }, [userInfo]);
 
+  useEffect(() => {
+    const showInstallAppModal = () => {
+      const isMobile = deviceInfo.isMobile;
+      const modalShown = localStorage.getItem('installAppModalShown');
+      const isPWA = window.matchMedia('(display-mode: standalone)').matches;
+      const isInstalled = localStorage.getItem('isAppInstalled');
+
+      if (isMobile && !isPWA && !modalShown && !isInstalled) {
+        //TODO: add modalshown key in localstorage
+        localStorage.setItem('installAppModalShown', 'true');
+        onOpen();
+      }
+    };
+
+    setTimeout(() => {
+      showInstallAppModal();
+    }, 1000);
+  }, [userInfo]);
+
+  const installApp = async () => {
+    if (installPrompt.current) {
+      // @ts-expect-error --> to remove prompt doesnt exist error
```

Comment:
we generally don't encourage `ts-ignore`s and `ts-expect-error`s
so please find a proper typesafe alternative